svthalia/concrexit
# Create personal Thalia Pay permissions

Closes #1257 and closes #1258 and closes #947

### Summary
Every user profile now also has a boolean (default True) whether that member can use Thalia Pay or not. Only members that have that set to True can actually use and see Thalia Pay. People that are allowed to use Thalia Pay, but do not have a valid bank account, will see disabled Thalia Pay buttons with a popover instructing them how to start using it.

If users are disallowed to use Thalia Pay, all buttons will disappear. And still, when the global setting `THALIA_PAY_ENABLED` is disabled, paying will never be possible.

Also added two actions for the user admin

### How to test
1. Check the boolean field in user profiles
2. Check the 2 actions in the user admin
3. Check out the Thalia Pay button everywhere
